modules/store/testing/src/mock_store.ts

```diff
@@ -18,9 +18,13 @@ import { MOCK_SELECTORS } from './tokens';
 if (typeof afterEach === 'function') {
   afterEach(() => {
     try {
-      const mockStore: MockStore | undefined = TestBed.inject(MockStore);
-      if (mockStore) {
-        mockStore.resetSelectors();
+      const TestBed = getTestBed() as any;
```

Comment:
Add a comment here if why this is needed, so we'll have some context later when we can potentially remove it.

Reply:
Ok, not sur if my english is good enough, but I tried ...